### DISCUSSION TITLE
hello i am new

### DISCUSSION THREAD

**Discussion by ErenKaya31** [YoshiCrafter29/YoshiCrafterEngine#98] [General]
hello i wanted to see what is yoshi engine and i was excited and i love it so its good :) :+1:

**Comment by ErenKaya31** [comment]
so i leaved psych engine YEET

**Reply by ghost** [reply]
welcome ( i just joined too)

**Reply by ErenKaya31** [reply]
hey @olliesus11131 did you leave psych engine too?

**Reply by ghost** [reply]
im not sure if im leaving for good but maybe

**Reply by ghost** [reply]
im testing stages
